runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/config/PercentEncodableStringComparator.java

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright 2021-2023 Aklivity Inc.
+ *
+ * Aklivity licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+package io.aklivity.zilla.runtime.binding.http.internal.config;
+
+import java.util.Comparator;
+
+public class PercentEncodableStringComparator implements Comparator<String>
```

Comment:
Apologies for the dueling naming feedback.
Lets rename this to `HttpQueryStringComparator`. (all classes in this project start with `Http`)

Reply:
OK